src/controller/order.controller.ts

```diff
@@ -162,15 +170,16 @@ class OrderController {
         }
     }
 
-    async getAllOrders(req, res, next) {
+    async getAllOrders(req: CustomRequest<null, null, GetAllOrders, null>, res: Response, next: NextFunction) {
         try {
             let {limit, offset, masterId, userId} = req.query
-            if (limit > 50) limit = 50
-            if (!offset) offset = 0
+            if (limit && +limit > 50) limit = '50'
+            if (!offset) offset = '0'
             let where
-            if (userId) where = {'userId': userId}
-            else if (masterId) where = {'masterId': masterId}
-            const options = {
+            if (userId) where = {'userId': +userId}
+            else if (masterId) where = {'masterId': +masterId}
+
+            const options: any = {
```

Comment:
You should import from sequelize interface of payload of findAndCountAll methode